steps/synopsys-dc-synthesis/scripts/compile.tcl

```diff
@@ -69,7 +69,7 @@ if { $dc_uniquify_with_design_name == True } {
 
 # Use naming rules to preserve structs
 
-define_name_rules verilog -preserve_struct_ports
+define_name_rules verilog -preserve_struct_ports -case_insensitive
```

Comment:
I think the naming case sensitivity problems came because DC uses uppercase N's and Innovus uses lowercase n's for nets (e.g., N35/n35). We didn't need to tell DC to do this... just needed Innovus update_names at the end. It is good to have this anyway though.

Reply:
Well then it is even better to do it already in DC. This way the names in the reports and logs before the Innovus signoff step will be the same as in the final netlist.

Reply:
Yep, perfect!